protocol/implementations/js/src/types.ts

```diff
@@ -0,0 +1,32 @@
+export namespace TbDex {
+  export type Offering = {
+    placeholder: string;
```

Comment:
did you want to use all the fields we just changed per the readme changes in https://github.com/TBD54566975/tbdex-protocol/pull/77, or do that later?

Reply:
down to go either way! could include them in this PR orrr could merge this to main and do a fast-follow PR with the 🥩

Reply:
let's do a fast follow PR just to keep PRs _atomic_ :p